lib/canvas/live_events.rb

```diff
@@ -184,6 +205,23 @@ def self.wiki_page_deleted(page)
     })
   end
 
+  def self.attachment_created(attachment)
+    post_event_stringified('attachment_created', get_attachment_data(attachment))
+  end
+
+  def self.attachment_updated(attachment, old_display_name)
+    payload = get_attachment_data(attachment)
+    if old_display_name
```

Comment:
I was just following the pattern for updated events. We don't really have a need for knowing what the old display name was, we just need to be notified that an attachment has a new one.

I'm not all that familiar with Ruby or Rails, but would it make sense to provide the `changes` hash for `update` events? That way you're not cherry picking which attributes are getting carried over. Again, I'm not familiar with whether there's sensitive data on any of the models that have an `update` event though.